rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -185,6 +192,23 @@ object RudderProperties {
       case FileSystemResource(file) => ConfigFactory.load(ConfigFactory.parseFile(file))
     }
   }
+
+  //some logic for the authentication providers
+  val authenticationMethods: Seq[AuthenticationMethods] = {
+    val names = try {
+      //config.getString can't be null by contract
+      config.getString("rudder.auth.type").split(",").toSeq.map( _.trim).collect { case s if(s.size > 0) => s}
```

Comment:
Is the property fetched from the configuration.property file ? if so it misses the updated configuration file  (and we should have a migration script)